lib/spotify/delete-playlist.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Spotify API Unfollow Playlist Endpoint
+ * https://developer.spotify.com/documentation/web-api/reference/#/operations/unfollow-playlist
+ * DELETE /playlists/{playlist_id}/followers
+ *
+ * deleteSpotifyPlaylist
+ * params: playlistId
+ *
+ * returns whether the playlist was successfully deleted
+ */
+
+import fetch from 'node-fetch';
+import { User } from '..';
+
+export const deleteSpotifyPlaylist = async (playlistId : string): Promise<boolean | null> => {
+  const root = await User.fromId('0');
+  if (!root) return false;
+
+  const accessToken = await root.getAccessToken();
+  const bodyParams = {
+    name: '(DELETE ME)',
+  };
+  const rename_res = await fetch(`https://api.spotify.com/v1/playlists/${playlistId}`, {
+    method: 'PUT',
+    headers: {
+      'Content-Type': 'application/json',
+      'Authorization': 'Bearer ' + accessToken,
+      'Host': 'api.spotify.com',
+    },
+    body: JSON.stringify(bodyParams),
+  });
+  if (!rename_res) return false;
```

Comment:
why do we need to rename it? (thought it might be a good idea to add this to the patch endpoint)!

Reply:
uhh its cuz unfollow doesn't actually delete the playlist from the cl account so I made it rename the playlist to be deleted

Reply:
working on that rn

Reply:
renaming was cuz i thought deleting wasnt working but it seems ok now